bitnami/discourse/templates/service.yaml

```diff
@@ -34,6 +34,6 @@ spec:
       nodePort: null
       {{- end }}
     {{- if .Values.service.extraPorts }}
-    {{- include "common.tplvalues.render" (dict "value" .Values.service.extraPorts "context" $) | indent 4 }}
+    {{- toYaml .Values.ingress.extraPorts | nindent 4 }}
```

Comment:
```suggestion
   {{- include "common.tplvalues.render" (dict "value" .Values.service.extraPorts "context" $) | nindent 4 }}
```

This change should be enought

Reply:
Is this solution just to switch indent with nindent?

Reply:
UPdated with bumped chart version

Reply:
Yes changing indent by nindent  works